src/views/HomeView.vue

```diff
@@ -39,29 +39,41 @@
               variant="outlined"
               color="primary"
               class="mt-4"
-              :small="$vuetify.display.mdAndDown ? true : false"
               to="/rules"
               data-cy="rules-link"
+              v-bind="{
+                ...($vuetify.display.mdAndDown ? {
+                  size: 'small',
+                } : {})
+              }"
```

Comment:
Like
```suggestion
              v-bind="{
                  size: $vuetify.display.mdAndDown ? 'small' : 'medium',
                } "
```

Reply:
This is less flexible when it comes to potentially configuring multiple props based on the same size, but we're only doing one here and I think this might be more legible. Thoughts?